web/src/app/core/services/ping.service.ts

```diff
@@ -33,4 +33,16 @@ export class PingService {
           .valueChanges())
       );
   }
+
+  public findAllPingsCount(projectUid: string): Observable<PingModel[]> {
+    return this.activityService
+      .start()
+      .pipe(
+        switchMap(() => this.afs
+          .collection<ProjectModel>('projects')
+          .doc<ProjectModel>(projectUid)
+          .collection<PingModel>('pings')
+          .valueChanges())
```

Comment:
As pings collection could get large, which will cause the frontend to get slow (possibly expensive), I think it would be better if we use the  `successful`/`unsuccessful` for each monitor in the UI and just total these.